src/lib/components/AgGrid.react.js

```diff
@@ -458,1559 +439,153 @@ DashAgGrid.propTypes = {
-     * Default Value: false
-     */
-    accentedSort: PropTypes.bool,
-
-    /**
-     * Set to 'ctrl' to have multi sorting work using the Ctrl or Command (for Apple)
-     * keys. See Multi Column Sorting.
-     */
-    multiSortKey: PropTypes.any,
-
-    /**
-     * Set to true to suppress sorting of un-sorted data to match original row data.
-     * See Big Data Small Transactions.
-     * Default Value: false
-     */
-    suppressMaintainUnsortedOrder: PropTypes.bool,
-
-    /**
-     * Set to true to override the default tree data filtering behaviour to instead exclude
-     * child nodes from filter results. See Tree Data Filtering.
-     * Default Value: false
-     */
-    excludeChildrenWhenTreeDataFiltering: PropTypes.bool,
-
-    /**
-     * Type of Row Selection.
-     */
-    rowSelection: PropTypes.any,
-
-    /**
-     * Set to true to allow multiple rows to be selected using single click. See Multi
-     * Select Single Click.
-     * Default Value: false
-     */
-    rowMultiSelectWithClick: PropTypes.bool,
-
-    /**
-     * If true then rows will not be deselected if you hold down Ctrl and click the row
-     * or press Space.
-     * Default Value: false
-     */
-    suppressRowDeselection: PropTypes.bool,
-
-    /**
-     * If true, row selection won't happen when rows are clicked. Use when you want checkbox
-     * selection exclusively.
-     * Default Value: false
-     */
-    suppressRowClickSelection: PropTypes.bool,
-
-    /**
-     * If true, cells won't be selectable. This means cells will not get keyboard focus
-     * when you click on them.
-     * Default Value: false
-     */
-    suppressCellSelection: PropTypes.bool,
-
-    /**
-     * Set to true to enable Range Selection.
-     * Default Value: false
-     */
-    enableRangeSelection: PropTypes.bool,
-
-    /**
-     * Set to true to enable Range Handle
-     * Default Value: false
-     */
-    enableRangeHandle: PropTypes.bool,
-
-    /**
-     * Set to true to enable Fill Handle
-     * Default Value: false
-     */
-    enableFillHandle: PropTypes.bool,
-
-    /**
-     * Set to 'x' to force the fill handle direction to horizontal, or set it to 'y'
-     * to force the fill handle direction to vertical
-     * Default Value: xy
-     */
-    fillHandleDirection: PropTypes.string,
-
-    /**
-     * Set it to true to prevent cell values from being cleared when the Range Selection
-     * is reduced by the Fill Handle.
-     * Default Value: false
-     */
-    suppressClearOnFillReduction: PropTypes.bool,
-
-    /**
-     * Set to true to enable Managed Row Dragging.
-     * Default Value: false
-     */
-    rowDragManaged: PropTypes.bool,
+    defaultColDef: PropTypes.object,
 
     /**
-     * Set to true to suppress Row Dragging.
-     * Default Value: false
+     * Sets the Row Model type.
+     * Default Value: ['clientSide', 'infinite', 'viewport', 'serverSide']
```

Comment:
```suggestion
     * Default Value: 'clientSide'
```
I know this isn't new in this PR, just a git artifact... but it drew my attention.